components/atoms/Label/Label.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { styled } from 'nativewind';
+import { Text as NativeText } from 'react-native';
```

Comment:
Komponent Text już jest gotowy w components/atoms/Text więc skorzystaj z niego

Reply:
to pytanie czy tej labelki potrzebujemy? :) bo robotę robi już component Text :) no chyba ze zaprojektowałeś aby component Text od labelki czym się wyróżniał? bo rozuemiem że jeli labelka ma korzystac z Text to będzie jakimś bardziej rozbodowanym atomem?